data/json/recipes/recipe_deconstruction.json

```diff
@@ -692,8 +692,10 @@
     "reversible": true,
     "qualities": [ {"id": "SCREW","level": 1 } ],
     "tools": [
-      [ [ "soldering_iron", 20 ] ],
-      [ [ "toolset", 20 ] ]
+     [
+      [ "soldering_iron", 20 ],
+      [ "toolset", 20 ]
+     ]
```

Comment:
[like this ?](https://github.com/CleverRaven/Cataclysm-DDA/pull/16944/commits/f9b7a3d555ed70815db0acd12134f4a58a12885e#diff-a1de6fb21e57d79260285970ccdc28e6R694)